phpstan-baseline.neon

```diff
@@ -2277,7 +2277,7 @@ parameters:
 
 		-
 			message: "#^Anonymous function should have native return typehint \"string\"\\.$#"
-			count: 5
+			count: 6
```

Comment:
you should add native return typehint string to the function instead

Reply:
done